website/community/syncs.md

```diff
@@ -7,32 +7,35 @@ last_modified_at: 2020-09-01T15:59:57-04:00
 
 # Community Syncs
 
-We have setup the following regular syncs for community users and developers to meet, interact and exchange ideas. 
-Meetings will be recorded and made available, on a best-effort basis.
+We have setup the following regular syncs for community users and developers to meet, interact and exchange ideas. Meetings will be recorded and made available, on a best-effort basis.
 
 See more informations about Chinese community sync call by changing the website language into Chinese.
 
 ## Monthly Community Call
 
-**[ZOOM LINK TO JOIN](https://zoom.us/j/96743829685?pwd=S3pxRVVKTDljWVFVWW1IU1UvY0JEUT09)**
+Participate in our community calls by joining us on LinkedIn Live every month on the **Fourth Wednesday**, at 09:00 AM Pacific Time (US and Canada)([translate to other time zones](https://www.worldtimebuddy.com/?qm=1&lid=5368361,2643743,1264527,1796236&h=2643743&date=2022-5-25&sln=17-18&hf=1)). To join, click on the image right below and join the community call on the events page:
```

Comment:
Can we also add the link to join explicitly ( Linkedin Events page URL) in addition to the image linked to the URL ?

Reply:
sure